doc-surrealdb_versioned_docs/version-1.x/faqs/known-issues.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+sidebar_label: Known Issues
+title: Known Issues
+description: The purpose of this document is to hightlight some of the known issues with SurrealDB. This will help you to understand the current limitations and issues that are being worked on.
+---
+
+# Known Issues
+
+Here are some known bugs and issues with SurrealDB:
+
+## SDKs
+
+- Some query functionality is not yet available in the SDKs. Such as `IF EXISTS` and `IF NOT EXISTS` clauses in the [DEFINE statement](/docs/surrealdb/surrealql/statements/define/). This means that if you used these clauses in your query method(as seen in the example below), the SDK would not be able to parse the query correctly.
```

Comment:
Maybe change to a comma here:

...in the SDKs, such as...

method(as seen -> method (as seen